lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -61,6 +61,7 @@ def self.canonicalize_url(driver_url, config)
 
     # Load balancer methods
     def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
+      lb_options ||= {}
```

Comment:
I prefer to see `lb_options = {}` in the method signature, but thats pretty personal.